tasks/homework.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+)
+
+type ZooKeeper struct {
+	Name string
+}
+
+func (keper *ZooKeeper) SendToCage(animal *Animal, cage *Cage) {
+	cage.Animal = animal
+	animal.IsFree = false
+}
+
+type Animal struct {
+	Name   string
+	IsFree bool
+}
+
+func (a *Animal) makeFree() {
+	a.IsFree = true
+}
+
+const (
+	Lion   = "Lion"
+	Tiger  = "Tiger"
+	Duck   = "Duck"
+	Monkey = "Monkey"
+)
```

Comment:
Чи можемо ми використати typed constant тут?
```suggestion
type AnimalType string

const (
	Lion   AnimalType = "Lion"
	Tiger  AnimalType = "Tiger"
	Duck   AnimalType = "Duck"
	Monkey AnimalType = "Monkey"
)
```

Про константи можна почитати тут https://go.dev/doc/effective_go#constants і тут https://go.dev/blog/constants

Reply:
так, замінив